### PR TITLE
Il/remove attachments

### PR DESCRIPTION
**Pull request by ilodidoj**
[Link to ZenHub issue.](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/885)

## What does this change?
Allow intake specialist to remove attachment from a report. 

Note, we are setting the Active=False to instead of real file delete.
Link will not display on the report once removed successfully, aka Active=False in the model. Original uploaded file will be preserved in designated storage. 

### Author

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [x] Check for, document, and establish a testing plan for any behavior that may vary across environments or is otherwise difficult to test.
+ [x] Check for [accessibility](/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [x] Check for any behavior that may vary across environments or is difficult to test, and ensure that it is well-understood, documented, and that there is a testing plan in place.
+ [x] Re-check for [accessibility](/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

**Comment by ilodidoj** [comment]
all functionality and test are ready with the exception of attachment "Trash" button icon.

File: run.sh [hunk 1]
```diff
@@ -11,7 +11,13 @@ node node_modules/gulp/bin/gulp build-sass
 # If LOCALSTACK is set in environment, this will upload static files to the localstack s3 service running in docker
 # Otherwise the development server is handling static files
 if [[ -n "${USE_LOCALSTACK}" ]]; then
+    echo Removing crt-portal s3 bucket
```
Comment (timoballard) [inline run.sh:14]: We've been seeing some inconsistent behavior with respect to the static files updating/not updating in the localstack s3 bucket when running locally, which can frustrate development and debugging efforts. This is fairly blunt, but effective at ensuring that you have the latest stuff in your static bucket for local development and testing

File: crt_portal/cts_forms/templates/partials/attachment-removal-confirmation-modal.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        </h1>
+      </div>
+      <div class="modal-form">
+        <p>
+          {% trans "Are you sure you want to remove the following attachment?" %}
+          <h3 id="attachment-removal--filename">
+          </h3>
+        </p>
+        <div class="modal-footer">
+          <a id="attachment-removal--no" href="#" class="usa-button usa-button--outline button--cancel light-button">{% trans "No" %}</a>
```
Comment (Jkrzy) [inline crt_portal/cts_forms/templates/partials/attachment-removal-confirmation-modal.html:18]: The text here is rendering in Red when the element is active/focused.
Screen Shot 2021-03-15 at 11 24 41 AM
?

**Review by Jkrzy — CHANGES_REQUESTED** [review]
Thanks @ilodidoj, looking great! I left a few comments throughout where there's some open questions/areas for improvement.

File: crt_portal/cts_forms/views.py [hunk 3]
```diff
@@ -712,6 +712,22 @@ def post(self, request, report_id):
         return redirect(url)
 
 
+class RemoveReportAttachmentView(LoginRequiredMixin, FormView):
+    def post(self, request, attachment_id):
+        logger.info(f'User {request.user} removing attachment with id {attachment_id}')
```
[the remark this answers is not in the document]
Comment (ilodidoj) [reply crt_portal/cts_forms/views.py]: moved as suggested.

File: crt_portal/cts_forms/templates/partials/attachment-removal-confirmation-modal.html (hunk 2 above)
Comment (ilodidoj) [reply crt_portal/cts_forms/templates/partials/attachment-removal-confirmation-modal.html]: made changes to css to fix this.

File: crt_portal/cts_forms/templates/partials/attachment-removal-confirmation-modal.html [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  <div class="modal-wrapper" role="dialog" aria-modal="true" aria-describedby="modal_dialog_header">
+    <div class="modal-content modal-content--small">
+      <div class="modal-header">
+        <h1 id="modal_dialog_header" class="h2__display">
+          {% trans "Remove Attachment" %}
+        </h1>
+      </div>
+      <div class="modal-form">
+        <p>
+          {% trans "Are you sure you want to remove the following attachment?" %}
```
[the remark this answers is not in the document]
Comment (ilodidoj) [reply crt_portal/cts_forms/templates/partials/attachment-removal-confirmation-modal.html]: translation template tag removed as suggested.

**Review by Jkrzy — APPROVED** [review]
Thanks @ilodidoj this looks good!

Please run through the PR checklist before we merge.

**Pull request merged**